lib/galaxy/webapps/galaxy/api/__init__.py

```diff
@@ -410,3 +412,58 @@ async def try_get_request_body_as_json(request: Request) -> Optional[Any]:
         body = await request.json()
         return body
     return None
+
+
+search_description_template = Template(
+    """A mix of free text and Github-style tags used to filter the index operation.
```

Comment:
```suggestion
    """A mix of free text and GitHub-style tags used to filter the index operation.
```

Reply:
Fair - sorry about that and thanks for the review!